### DISCUSSION TITLE
Hide (some) postprocessors from console?

### DISCUSSION THREAD

**Discussion by aprilnovak** [idaholab/moose#16815] [Q&A General] [answered]
Hi all,

Is it possible to hide individual postprocessors from being output to the screen? For instance, if all I care about is the value of `LinearCombinationPostprocessor`, that combines multiple other postprocessors into something meaningful, is it possible to hide the postprocessors that just went _into_ the calculation of the `LinearCombinationPostprocessor`? This would make it easier to monitor the solution progression so I don't have loads of screen output.

I know I can hide them from the Exodus file with `hide`- is there something similar for the screen output?

Thanks!

-April

**Comment by hugary1995** [comment, marked as answer]
Does this work?
```
[Outputs]
  [foo]
    type = Console
    hide = 'bar baz'
  []
[]
```

**Reply by aprilnovak** [reply]
Yes! Wonderful, thank you :)

**Reply by aeslaughter** [reply]
You can also put "outputs = none" in the PP/VPP block as well.
